src/vue/forms/TransferForm.vue

```diff
@@ -0,0 +1,525 @@
+<template>
```

Comment:
Very fat form (~500 rows). The dashboard page developing took very much time, but we still need to reduce the mess here. I’d prefer create a separate task to refactor this form